Tests/TestSuites/MapNodeTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import XCTest
+@testable import StencilSwiftKit
+
+class MapNodeTests: XCTestCase {
+  static let context = [
+    "items1": ["one", "two", "three"],
+    "items2": ["hello", "world", "everyone"]
+  ]
+  
+  func testBasic() {
+    let template = SwiftTemplate(templateString: Fixtures.string(for: "map-basic.stencil"), environment: stencilSwiftEnvironment())
+    let result = try! template.render(MapNodeTests.context)
+    
+    let expected = Fixtures.string(for: "map-basic.out")
+    XCTDiffStrings(result, expected)
```

Comment:
I'll take a look at the Stencil tests.

I'll have to try and convert the current tests for this (and other nodes) to comparable tests using constructors and such.